package.json

```diff
@@ -12,13 +12,16 @@
     "start": "next start -p 80"
   },
   "nyc": {
-    "reporter": ["html"]
+    "reporter": [
+      "html"
+    ]
```

Comment:
No big deal, but please be careful not to commit unrelated changes that don't mean anything. I'm guessing this change is caused by some auto-formatting feature in your editor. It should not have been committed.

Reply:
That's very strange since I've turned off all auto-formatting, but either way I shouldn't have made it part of the commit, sorry about that.